core/static/js/index.js

```diff
@@ -0,0 +1,17 @@
+window.onload=function() {
+  document.getElementById("zipcode").focus();
+}
+
+function zipup() {
+  var zipcode = document.getElementById('zipcode').value;
+  if (zipcode.search(/[abcdefghijklmnopqrstuvwxyz]/i) != -1) {
```

Comment:
seems like we could refine this search to only match 0-9 instead of not matching other characters.